Gemfile

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
There is a Gemfile in chef-powershell/Gemfile. I think this one should be removed.

Reply:
That gemfile applies to chef-powershell only.

Reply:
ok. I'd think one needs to be removed. This PR should only include one Gemfile unless I'm missing something.

Reply:
deleted